src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx

```diff
@@ -98,25 +98,12 @@ export const SettingsView = (props: Props) => {
               <div className={styles.cancelSection}>
                 <h3>{l10n.getString("settings-cancel-plus-title")}</h3>
                 <p>{l10n.getString("settings-cancel-plus-details")}</p>
-                {props.enabledFeatureFlags.includes("CancellationSurvey") ? (
-                  <CancelFlow fxaSubscriptionsUrl={props.fxaSubscriptionsUrl} />
-                ) : (
-                  <TelemetryLink
-                    href={props.fxaSubscriptionsUrl}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    eventData={{
-                      link_id: "cancel_plus",
-                    }}
-                  >
-                    {l10n.getString("settings-cancel-plus-link-label")}
-                    <OpenInNew
-                      alt={l10n.getString("open-in-new-tab-alt")}
-                      width="13"
-                      height="13"
-                    />
-                  </TelemetryLink>
-                )}
+                <CancelFlow
```

Comment:
With CD to stage enabled, could we maybe have two flags, `CancellationFlow` and `ConfirmCancellation`? Just to make sure we have some control over when this gets shown to all users, and not make people hesitant about deploying things to prod.

(So revert the changes in this file, renaming `CancellationSurvey` to `CancellationFlow`, and then adding back the `confirmationFlagEnabled` prop?)